TsdrApi.php

```diff
@@ -334,6 +374,7 @@ private function _cleanInput($userInput) {
 	  $userInput = trim($userInput);
 	  $userInput = stripslashes($userInput);
 	  $userInput = htmlspecialchars($userInput);
+	  $userInput = str_replace(",", "", $userInput);
```

Comment:
What's this for? Is this necessary?

Reply:
If the user enters commas in a registration or serial number it fails without this line.